### DISCUSSION TITLE
Housing Delivery Test

### DISCUSSION THREAD

**Discussion by MrMikeRose** [digital-land/data-standards-backlog#91] [Planning consideration]
# Housing delivery test

This is data centrally compiled by MHCLG to assess whether LPAs are meeting their housing targets and needs. It is [published annually on GOV.UK](https://www.gov.uk/government/collections/housing-delivery-test).

We think this is a good consideration to look at because:

* data already exists, it is published yearly
* the data is collated, maintained and published by MHCLG
* it is a good entry point into the [housing numbers](https://design.planning.data.gov.uk/planning-consideration/housing-numbers) space because it touches on housing need and delivery, both on a local and national level

**Comment by colmjude** [comment]
## Available data

The data is [published annually on GOV.UK](https://www.gov.uk/government/collections/housing-delivery-test). Users can download it as a spreadsheet.

The data is available under OGL
